src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -717,4 +799,16 @@ public void write(OutputStream output) throws IOException, WebApplicationExcepti
 			}
 		}
 	}
+
+	//Activate a scope with a span and return
+	public io.opentracing.Scope activateScope(String spanName, HttpHeaders httpHeaders) {
```

Comment:
This seems like it could be a good addition to our utils package, perhaps as some sort of common-utils module.